test/test_source_again.rb

```diff
@@ -0,0 +1,24 @@
+require 'base_test'
+
+class TestSourceTypeAgain < Test::Unit::TestCase
+  include TestSourceBase
+  extend VariableAccessor
+  include VariableClassAccessor
+
+  def self.startup
+    @btx_variables = {
+      btx_model_path: './test/cases_source_again/1.btx_model.yaml',
+      btx_target_log_path: './test/cases_source_again/1.btx_log.txt',
+      btx_component_type: 'SOURCE',
+      btx_component_name: 'source',
+      btx_pluggin_name: 'metababel_tests',
+      btx_callbacks_path: './test/cases_source_again/1.callbacks.c',
+      btx_component_path: './test/SOURCE.metababel_test'
+    }
+  end
+
+  def subtest_generate_source_callbacks
+	FileUtils.cp(btx_variables[:btx_callbacks_path],btx_variables[:btx_component_path])
```

Comment:
Consider to use `assert_nothing_raised` as shown below. It may seem redundant, and do not display the whole stack trace information as [she argues](https://thagomizer.com/blog/2013/06/11/assert-nothing-raised.html).  but look at the following examples in our code (images below).
 
```ruby
assert_nothing_raised do 
  FileUtils.cp(btx_sink_variables[:btx_callbacks_path],btx_sink_variables[:btx_component_path])
end
```
If we use `assert_nothing_raised` we will be able to locate the error an will have specific information about it. 
![image](https://user-images.githubusercontent.com/9855594/229546481-9eee8490-94bf-4d63-991b-7c80bba05e75.png)

On the other hand, if we **do not use it** we will end up with a lot of stuff, and if you see, the useful information is at the end.
![image](https://user-images.githubusercontent.com/9855594/229548585-063c08e7-e44b-4242-90a0-8c83b26de31c.png)

Reply:
Ok let me do something smarter